packages/ui-components/src/Icon/Illustration.module.scss

```diff
@@ -0,0 +1,4 @@
+.illustration {
+  height: 152px;
```

Comment:
@nathanstilwell , can it be useful to make illustrations more responsive? My concern is about cases when page is resized to smaller sizes. I would suggest 
- to use maxHeight and maxWidth 
- or allow illustrations to fill entire parent's container with preserved image proportions

Reply:
That's a good idea. We might as well use the responsive nature of these SVGs. Given where we use these illustrations (usually in the middle of an empty page or section) I don't believe we would see this issue very often, but it's good to be forward thinking.